Sources/PackageModel/UserToolchain.swift

```diff
@@ -499,6 +502,18 @@ public final class UserToolchain: Toolchain {
         self.isSwiftDevelopmentToolchain = false
         #endif
 
+        if let customInstalledSwiftPMConfiguration = customInstalledSwiftPMConfiguration {
```

Comment:
What’s the use case for overriding the config file? Or: How do you even override it? I don’t see any constructor call with `customInstalledSwiftPMConfiguration`?

Reply:
It would be for a client of libSwiftPM that creates their own instance of `UserToolchain` and passes it to the workspace. I don't have any concrete use case in mind, but usually this is something that comes up later on.

Reply:
Makes sense. Thanks!